### PR TITLE
Extract keywords and context

### PR DESCRIPTION
**Pull request by akhilaananthram**
This uses cortical.io's API to get the keywords and context.

This is for: https://github.com/numenta/nupic.fluent/pull/109

@BoltzmannBrain

### CODE REVIEW

File: tests/integration/get_context_from_text_test.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    texts = ["The jaguar is a big cat", "a feline in the Panthera genus."]
+    bitmaps = [client.getTextBitmap(t)["fingerprint"]["positions"] for t in texts]
+    contexts = client.getContextFromText(bitmaps)
+    self._checkValidContexts(contexts)
+
+  def _checkValidContexts(self, contexts):
+    # Assert: check the result object.
+    self.assertIsInstance(contexts, list,
+      "Returned object is not of type list as expected.")
+    self.assertGreaterEqual(len(contexts), 1, "Returned object did not contain any elements")
```
Comment (BoltzmannBrain) [inline tests/integration/get_context_from_text_test.py]: wrap at 80 char

**Comment by BoltzmannBrain** [comment]
@akhilaananthram please clean up the formatting in the tests (see my comments in #30). Otherwise :+1:

**Comment by akhilaananthram** [comment]
@BoltzmannBrain: fixed formatting and line continuation

**Comment by BoltzmannBrain** [comment]
:+1:

**Pull request merged**